src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-		<Version>1.0.3-preview</Version>
+		<Version>2.0.0-preview</Version>
```

Comment:
Add package release notes

Reply:
@avneeshrai if you are in preview, you don't have to bump up major version.
Your new version can be 1.1.0-preview